expertsystem/state/particle.py

```diff
@@ -305,19 +305,21 @@ def to_dict(input_ordered_dict: OrderedDict) -> dict:
     name_label = Labels.Name.name
     with open(file_path, "rb") as xmlfile:
         full_dict = xmltodict.parse(xmlfile)
-        for particle_definition in full_dict["ParticleList"]["Particle"]:
-            particle_name = particle_definition[name_label]
-            particle_list[particle_name] = to_dict(particle_definition)
+        full_dict = full_dict.get("root", full_dict)
+    for particle_definition in full_dict["ParticleList"]["Particle"]:
+        particle_name = particle_definition[name_label]
+        particle_list[particle_name] = to_dict(particle_definition)
```

Comment:
shouldn't this be indented to be inside the with scope?

Reply:
Why?

Reply:
Oh i thought with would create a scope, but it doesn't. Nvm then